### DISCUSSION TITLE
Version 4.0.4 Changelog

### DISCUSSION THREAD

**Discussion by Wapp92** [OnionUI/Onion#693] [Q&A]
Hi, i have the version 4.0.3 install. With a butch o f custum cheats and different path configure.

I was wondering if there is significant change in the new version 4.0.4 ? 

The release says, it make version 4.0.4 compatible with new firmware. Is that all ?

**Comment by Aemiii91** [comment]
Yes, that is the only change. Take a look at the [V4.1.0-alpha6](https://github.com/OnionUI/Onion/releases/tag/latest) release if you're interested in trying out the latest features.
